extensions/wikia/AdEngine/js/tracking/adInfoListener.js

```diff
@@ -69,12 +67,8 @@ define('ext.wikia.adEngine.tracking.adInfoListener',  [
 		var slotFirstChildData = slot.container.firstChild.dataset,
 			pageParams = JSON.parse(slotFirstChildData.gptPageParams || '{}'),
 			slotParams = JSON.parse(slotFirstChildData.gptSlotParams || '{}'),
-			slotPricesIgnoringTimeout = bidders && bidders.isEnabled()
-				? bidders.getCurrentSlotPrices(slot.name)
-				: lookupServices.getCurrentSlotPrices(slot.name),
-			realSlotPrices = bidders && bidders.isEnabled()
-				? bidders.getDfpSlotPrices(slot.name)
-				: lookupServices.getDfpSlotPrices(slot.name),
+			slotPricesIgnoringTimeout = bidders.getCurrentSlotPrices(slot.name),
```

Comment:
`bidders` can be not defined / not enabled

Reply:
Agree, fixed.